contributor-docs/release-guide.md

```diff
@@ -1228,6 +1228,24 @@ Ping [dev@](mailto:dev@beam.apache.org) mailing list for assistance if you need
 
 Copy the source release from the `dev` repository to the `release` repository at `dist.apache.org` using Subversion.
 
+```
+svn co https://dist.apache.org/repos/dist/dev/beam dev  # Checkout the `dev` artifact repo.
+
+svn co https://dist.apache.org/repos/dist/release/beam release  # Checkout the `release` artifact repo.
+
+mkdir release/$RELEASE_VERSION
+
+# Copy files from the `dev` artifact repo to the `release` artifact repo.
+
+cd release
+
+svn add $RELEASE_VERSION
+
+svn rm $OLD_RELEASE_VERSION   # Delete all artifacts from old releases.
```

Comment:
Oh, it looks like maybe this didn't work this time as well? https://dist.apache.org/repos/dist/dev/beam/ still has a version for 2.52.0 (and a couple other versions)

Reply:
This is to remove old artifacts from the release repo. Removing from the dev repo can be done by the release manager when needed (or from any committer).